README.old.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
```

Comment:
Great job @miltonHenschel 🤝

- [ ] [Optional] Could you fix the broken MIT license link to make your readme file more professional?
- [ ] [Optional] You may also remove the second author and customize the remaining author using your credentials.

Reply:
Thanks @1GORDON ! I'll do it.